README.md

```diff
@@ -309,6 +309,7 @@ require('crates').setup {
         enabled = false,
         name = "Crates",
     },
+    on_attach = function(bufnr) end,
```

Comment:
This will also be overwritten

Reply:
Pushed another fix for using `default_text` in `README.md`.